generator/01-header.yml.mustache

```diff
@@ -12,7 +12,14 @@ parameters:
   required: true
 - name: GITHUB_OAUTH_CLIENT_SECRET
   description: GitHub OAuth client secret
+  required: true{{#Probeless}}
```

Comment:
I'd like to rename the 'Probeless' parameter to something like 'CI' as this matches better the purpose (as its not only about 'probeless', which btw is probably not necessary anymore to disable probes for CI as we are now on OpenShift 3.5. for the CI environment)